src/pages/index.js

```diff
@@ -1,87 +1,239 @@
-cardPreview.setEventListeners();
+/*Functions*/
 
-function handleImageClick(name, link) {
-  cardPreview.open({ name, link });
+function openPreviewModal(data) {
+  previewImagePopup.open(data);
 }
 
-function createCard(cardData) {
-  const cardElement = new Card(cardData, "#card-template", handleImageClick);
-  return cardElement.getView();
+/* Instantiations*/
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d5d1d874-a161-48bd-b3b4-ae8e38400ed9",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardSection;
+
+api
+  .getUserInfo()
+  .then((res) => {
+    userInfo.setUserInfo(res);
+    userInfo.setUserAvatar(res);
+  })
+  .catch((err) => {});
+
+api
+  .getInitialCards()
+  .then((res) => {
+    cardSection = new Section(
+      { items: res, renderer: renderCard },
+      ".cards__list"
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+
+function renderCard(cardData) {
+  const card = new Card(
+    cardData,
+    selectors.cardTemplate,
+    openPreviewModal,
+    handleCardDeleteClick,
+    handleLikeClick
+  ).getCardElement();
+  cardSection.addItem(card);
 }
 
-const userInformation = new UserInfo({
+const userInfo = new UserInfo({
   name: ".profile__title",
   description: ".profile__description",
+  userAvatarSelector: ".profile__avatar",
 });
 
-const profileEditForm = new PopupWithForm("#profile-edit-modal", (data) => {
-  console.log("Form data", data);
-  userInformation.setUserInfo({
-    title: data.title,
-    description: data.description,
+const profileEditForm = new PopupWithForm(
+  "#profile-edit-modal",
+  handleProfileEditSubmit
+);
+const addCardPopup = new PopupWithForm("#add-card-modal", handleCardFormSubmit);
+const previewImagePopup = new PopupWithImage("#modal__preview-card");
+const confirmDeletePopup = new PopupWithConfirm("#delete-card-modal");
+
+const avatarImagePopup = new PopupWithForm(
+  "#edit-avatar-modal",
+  handleAvatarFormSubmit
+);
+
+/*Validation*/
+
+const editFormValidator = new FormValidator(config, profileForm);
+const addCardValidator = new FormValidator(config, cardForm);
+const avatarValidator = new FormValidator(config, avatarUpdateForm);
+
+/*Event Handlers*/
+
+function handleProfileEditSubmit(inputItems) {
+  profileEditForm.renderLoading(true);
+  api
+    .userProfileInfo(inputItems.name, inputItems.about)
+    .then((res) => {
+      console.log(res);
+      userInfo.setUserInfo(res);
+      profileEditForm.close();
+    })
+    .catch((err) => {
+      console.log(err);
+    })
+    .finally(() => {
+      profileEditForm.renderLoading(false);
+    });
+}
+function handleCardFormSubmit(data) {
+  addCardPopup.renderLoading(true);
+  api
+    .addNewCard(data.name, data.link)
+    .then((res) => {
+      renderCard(res);
+      addCardPopup.close();
+      cardForm.reset();
+      addCardValidator.resetValidation();
```

Comment:
`resetValidation` is for clearing validation errors but you don't have them on successful submition. Please, call `addCardValidator.disableButton();`